rest/api_test.go

```diff
@@ -1235,11 +1235,11 @@ func TestBulkDocsChangeToRoleAccess(t *testing.T) {
 	// Bulk docs with 2 docs.  First doc grants role1 access to chan1.  Second requires chan1 for write.
 	input := `{"docs": [
 				{
-					"_id": "bulk1", 
-				 	"type" : "roleaccess", 
-				 	"grantTo":"role:role1", 
+					"_id": "bulk1",
+				 	"type" : "roleaccess",
+				 	"grantTo":"role:role1",
 				 	"grantChannel":"chan1"
-				 }, 
+				 },
```

Comment:
I thought it was gofmt, but it looks like it's down to my editor settings. I'll turn it off for now.

I'm running [goimports](https://godoc.org/golang.org/x/tools/cmd/goimports) on save, but it will use the same formatting rules as gofmt. After reading into it a little more, gofmt will preserve any leading/trailing whitespace.